core/gnupg/gnupg.install

```diff
@@ -4,7 +4,7 @@ post_install() {
 }
 
 post_upgrade() {
-  if [[ $(vercmp $2 2.1.13-1) = -1 ]]; then
+  if [ $(vercmp $2 2.1.13-1) = -1 ]; then
```

Comment:
Sorry I forgot that a while back we switched from using bash to handle the install files to bourne shell where `[` would be correct, whereas with bash `[[` would have been correct.  Please ignore.